### ISSUE TITLE
Bugs on kl_loss of infom flow_occupancy loss

### ISSUE THREAD

**Issue by yixiaoshenghua** [chongyi-zheng/infom#2]
<img width="696" height="507" alt="Image" src="https://github.com/user-attachments/assets/84686506-d3ea-43dd-8fc4-3e864ec6391b" />
Dear authors, 
infom is a nice job. But I find some bugs of kl_loss computing of infom agent. Line 237 in Infom.py of agents misses pow(2) of jnp.exp(log_stds), is it right?
Thank you very much!

**Comment by yixiaoshenghua** [comment]
Thank you for your reply! I will try more experiments!
